foundation/storage/README.md

```diff
@@ -102,27 +108,73 @@ interface Encryptor {
 }
 ```
 
-There are configuration options for `SecretKeyEncryptor`:
+Configuration options for `SecretKeyEncryptor`:
 
 ```kotlin
- val encryptor = SecretKeyEncryptor {
+val encryptor = SecretKeyEncryptor {
     keyAlias = "TheKeyAlias"
-    enforceAsymmetricKey = true // Flag to enforce the use of an asymmetric key. default is false
-    secretKeyStorage = keyStorage // The storage for the secret key.
-    throwWhenEncryptError =
-        true // Flag to throw an exception when an error occurs during encryption. default is true
-    strongBoxPrefered = false // Flag to prefer StrongBox for key storage. default is true
+    enforceAsymmetricKey = true
+    secretKeyStorage = keyStorage
+    throwWhenEncryptError = true
+    strongBoxPrefered = false
```

Comment:
Typo - prefered -> preferred...
Good thing is that it appears to be only in the readme, and not in the actual code...

Reply:
Updated.